<launch>
    <group ns="vision">
			<node name="motionNode" type="motionNode" pkg="vision" respawn="False" output="screen" />
			<!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->
			
			<include file="$(find vision)/launch/general_params.launch" />	
			
			<!-- **************** /-->
			<!-- Debug Mode params /-->
			<!-- **************** /-->
			
				<!-- Enable Motion Debug /-->
				<param name="debugMotion" type="bool" value="True"/>
        
			<!-- **************** /-->
			<!-- Dummy Msg params /-->
			<!-- **************** /-->
				
				<!-- Enable Motion Dummy /-->
				<param name="motionDummy" type="bool" value="False"/>
        
            				
			<!-- *********************** /-->
			<!--  MotionDetector params  /-->
			<!-- *********************** /-->
					
				<!-- Test param for Motion Detection  /-->
				<param name="motionBuffer" type="int" value="5"/> 
				
				<!-- Test param for Motion Detection  /-->
				<param name="motionDiffThres" type="int" value="45"/>
				
				<!-- Test param for Motion Detection  /-->
				<param name="motionHighThres" type="double" value="4000"/>
				
				<!-- Test param for Motion Detection  /-->
				<param name="motionLowThres" type="double" value="380"/> 
				

    </group>
</launch>
